src/sentry/conf/server.py

```diff
@@ -929,7 +929,7 @@ def create_partitioned_queues(name):
 # XXX(dcramer): this doesn't work outside of a source distribution as the
 # webpack.config.js is not part of Sentry's datafiles
 SENTRY_WATCHERS = (
-    ('webpack', [os.path.join(NODE_MODULES_ROOT, '.bin', 'webpack'), '-d', '--watch',
+    ('webpack', [os.path.join(NODE_MODULES_ROOT, '.bin', 'webpack'), '--output-pathinfo', '--watch',
```

Comment:
`-d` is equivalent to `--devtool source-map --output-pathinfo` ... Removed the `--devtool` part because we declare in webpack.config.json, and they can clobber each other's values in non-expected ways.